packages/core/src/modules/proofs/ProofResponseCoordinator.ts

```diff
@@ -0,0 +1,86 @@
+import type { ProofRecord } from './repository'
+
+import { scoped, Lifecycle } from 'tsyringe'
+
+import { AgentConfig } from '../../agent/AgentConfig'
+
+import { AutoAcceptProof } from './ProofAutoAcceptType'
```

Comment:
Maybe we should have two types, like `AutoAcceptProof` for the holder and `AutoAcceptProofRequest` request for the issuer. Similarly with credentials. But that's of course more of a future improvement idea.

Reply:
Any particular reason for this? Maybe it allows for some use cases that I don't know of.

Reply:
I see 2 reasons:
The first is expressiveness. `AutoAcceptProof` means auto acceptance proof from the verifier point of view but also auto acceptance of the proof request from the holder's perspective. Even now, I'm not sure if I wrote it correctly.

The second is a potential use case when someone uses an agent for more than one role. Let's say I don't want to I want to accept proofs as a verifier but don't want to accept proof requests as a holder.